src/game_event.cpp

```diff
@@ -553,58 +549,24 @@ void Game_Event::Update() {
 	}
 	SetProcessed(true);
 
-	auto was_moving = !IsStopping();
-	Game_Character::UpdateMovement();
-	Game_Character::UpdateAnimation(was_moving);
-
-
-	if (starting && !Game_Map::GetInterpreter().IsRunning()) {
-		Game_Map::GetInterpreter().Setup(this);
-		Game_Map::GetInterpreter().Update();
-		running = true;
-	}
-
-	if (halting) {
-		running = false;
-		halting = false;
-	}
-}
-
-void Game_Event::UpdateParallel() {
-	int cur_frame_count = Player::GetFrames();
-
-	if (trigger != RPG::EventPage::Trigger_auto_start) {
-		// When this event becomes an auto-start event this frame through a
-		// refresh don't allow to run it.
-		// This prevents events  with lower IDs than the event doing the
-		// refresh from starting this frame.
-		frame_count_at_last_auto_start_check = cur_frame_count;
-	}
-
-	if (!data()->active || page == NULL || updating) {
-		return;
-	}
-
-	updating = true;
-
-	if (interpreter) {
+	if (trigger == RPG::EventPage::Trigger_parallel && interpreter) {
 		if (!interpreter->IsRunning()) {
 			interpreter->Setup(this);
 		}
 		interpreter->Update();
-	}
 
-	// Placed after the interpreter update because multiple updates per frame are allowed.
-	// This results in waits to finish quicker when an event collides with this event and
-	// emulates a RPG Maker bug)
-	// Only update the event once per frame
```

Comment:
just marking this here because this comment must be readded when this is fixed again

```
// Placed after the interpreter update because multiple updates per frame are allowed.
// This results in waits to finish quicker when an event collides with this event and 
// emulates a RPG Maker bug) 
```